src/corehost/build.sh

```diff
@@ -166,7 +166,7 @@ if [ -z $__commit_hash ]; then
 fi
 
 __build_arch_lowcase=$(echo "$__build_arch" | tr '[:upper:]' '[:lower:]')
-__base_rid=$__rid_plat-$__build_arch_lowcase
+export __base_rid=$__rid_plat-$__build_arch_lowcase
```

Comment:
@gkhanna79 Do you want to redefine __base_rid to CrossToolChainTargetRID ? or Do you want to define CrossToolChainTargetRID again here ? I just want to export the DistroRid which is a general purpose variable and use it in toolchain.cmake file. I don't understand why CrossToolChainTargerRID should be redefined even though it is the same as __base_rid.